app/views/devices_guidance/preparing_a_standard_windows_device.md

```diff
@@ -1,4 +1,4 @@
-This guidance is written with reference to standard Windows devices provided through the Get help with technology programme. It also applies to any new or refurbished devices a school or college has received through donations. Further information on [erasing data from donated devices](https://www.ncsc.gov.uk/blog-post/erasing-data-from-donated-devices) is available from the National Cyber Security Centre.
+This guidance is written with reference to standard Windows devices provided through the Get help with technology programme. It also applies to [restricted DfE laptops and tablets you have reset](/devices/guide-to-resetting-windows-laptops-and-tablets), and any new or refurbished devices a school or college has received through donations. Further information on [erasing data from donated devices](https://www.ncsc.gov.uk/blog-post/erasing-data-from-donated-devices) is available from the National Cyber Security Centre.
```

Comment:
Can you capitalise 'Standard' and 'DfE Restricted' for consistency please? Thanks

Reply:
@Lloyd-K done, cheers